package.json

```diff
@@ -14,6 +14,7 @@
     "debug": "~2.1.1",
     "express": "~4.11.1",
     "express-session": "^1.10.3",
+    "fastclick": "^1.0.6",
```

Comment:
Does this have anything to do with the logged-in issue? Or is this fixing #25 because I thought that was already fixed